ci: create benchmarks with automatic leader balancing

lenaschoenburg opened the pull request:
This cronjob previously existed in zeebe-benchmark-helm. Instead of adding it to camunda-load-test, we will simply create the job directly.

lenaschoenburg committed:
ci: create benchmarks with automatic leader balancing

This cronjob previously existed in zeebe-benchmark-helm.
Instead of adding it to camunda-load-test, we will simply create the job directly.

entangled90 reviewed (approved):
Thanks! 
I like the simpler approach :-D

lenaschoenburg committed:
fix: ignore false-positive shellcheck lint

It seems that shellcheck is not able to parse this yaml construct and
thinks that escaped newlines are needed.

lenaschoenburg committed:
fix: ignore when leader balancer already exists

lenaschoenburg committed:
fix: use different yaml multi-line to make shellcheck happy

Apparently shellcheck can't deal with `<` multi-line strings.
Disabling SC2215 didn't help either, so let's just bite the bullet and
use ugly escaped newlines.

ChrisKujawa commented:
@entangled90 @lenaschoenburg was it intended to adjust the old benchmark workflow? Instead the newer? https://github.com/camunda/camunda/blob/main/.github/workflows/camunda-load-test.yml

ChrisKujawa commented:
Ok 👍🏼 Yeah sorry, we kept the old file, as we ran into some issues. See for context https://camunda.slack.com/archives/C0807665N8G/p1760695609232739?thread_ts=1760021722.853319&cid=C0807665N8G
